app/src/main/res/layout-sw600dp/activity_default.xml

```diff
@@ -0,0 +1,41 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.design.widget.CoordinatorLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    android:id="@+id/root"
+    android:layout_width="match_parent"
+    android:layout_height="match_parent">
+
+    <RelativeLayout
+        android:id="@+id/crossfade_content"
```

Comment:
Make it a `FrameLayout` for lower performance impact, rename it in something like "drawerRoot" and move it into the exisitng `activity_default` file (Removing this file). It might not be needed for phones, but a `FrameLayout` is not expensive and I do not like a new resource folder for just this file. What do you think?